app/views/feeds/_form.html.erb

```diff
@@ -26,6 +26,9 @@
           <% end %>
         <% elsif megaphone_feed?(feed) %>
           <%= render "form_megaphone_config", podcast: podcast, feed: feed, form: form %>
+          <% if feed.persisted? %>
+            <%= render "form_audio_format", podcast: podcast, feed: feed, form: form, lock_format: true %>
```

Comment:
Ah, so they can see what format it'll be but not change it - +1.

Reply:
yeah, I thought this was a fairly decent way to do that, by passing in a value to the partial?
it works!